src/components/App/App.js

```diff
@@ -1,23 +1,32 @@
 import './App.css';
 import React, {Component} from 'react'
 import MovieContainer from '../MovieContainer/MovieContainer'
-import movieData from "../../movieMock.js"
+import { fetchAllMovies } from '../../apiCalls';
 import MovieInfo from '../MovieInfo/MovieInfo';
 import FooterForm from '../FooterForm/FooterForm';
 
 class App extends Component {
   constructor() {
     super()
     this.state = {
-      allMovies: movieData,
-      selectedMovie : null
+      allMovies: [],
+      selectedMovie : 0,
+      error : null
```

Comment:
I see the initial value of the error property of state is 'null'. Consider if there might be benefits, and what those benefits might be, to using an empty version of the expected data type instead of null.